src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/TrackingOptInModal.java

```diff
@@ -20,172 +20,180 @@
-        return urlBuilder.appendQueryStringToURL(urlBuilder.getUrlForPage(page), MODAL_INSTANT_GLOBAL);
-    }
-
-    public void isTrackingRequestsNotSend(List<String> elementsList,
-                                          NetworkTrafficInterceptor networkTrafficInterceptor) {
-        wait.forX(WAITNG_TIME_FOR_ALL_REQUESTS);
-        for(int i=0; i<elementsList.size(); i++){
-            Assertion.assertFalse(isSuccessfulResponseByUrlPattern(networkTrafficInterceptor, elementsList.get(i)),
-                    "Request to " + elementsList.get(i) + " services was found");
-        }
-    }
-
-    public void isTrackingRequestSend(List<String> elementsList,
-                                      NetworkTrafficInterceptor networkTrafficInterceptor) {
-        for(int i=0; i<elementsList.size(); i++){
-            wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor,elementsList.get(i));
-        }
-    }
-
-    public boolean areResponsesByUrlPatternSuccessful(List<String> elementsList,
-                                      NetworkTrafficInterceptor networkTrafficInterceptor) {
-        boolean result = true;
-        for(int i=0; i<elementsList.size(); i++){
-            try {
-                wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor,elementsList.get(i));
-            } catch (Exception e) {
-                PageObjectLogging.log("Did not get successfull response with element: " + elementsList.get(i),
-                        e, false);
-                result = false;
-            }
-        }
-        return result;
-    }
-
-    private boolean isSuccessfulResponseByUrlPattern(final NetworkTrafficInterceptor trafficInterceptor,
-                                                    final String pattern) {
-        try {
-            entry = trafficInterceptor.getEntryByUrlPattern(pattern);
-            return entry.getResponse().getStatus() < 400;
-        }catch (NullPointerException ex) {
-            return false;
-        }
-    }
-
-    private static String appendTrackingOptOutParameters(String url, String[] instantGlobals) {
-        UrlBuilder urlBuilder = new UrlBuilder();
-
-        for (String instantGlobal : instantGlobals) {
-            url = urlBuilder.globallyEnableGeoInstantGlobalOnPage(url, instantGlobal);
-        }
-        return url;
-    }
-
-    private String urlWithInstantGlobals(String[] instantGlobals, Page page){
-        String url = urlOptInModalDisplayedOasis(page);
-        String urlWithInstantGlobals = appendTrackingOptOutParameters(url, instantGlobals);
-        return urlBuilder.appendQueryStringToURL(urlWithInstantGlobals, MODAL_INSTANT_GLOBAL);
-    }
-
-    public void logTrackingCookieValue() {
-        PageObjectLogging.logInfo("Geo cookie: ", driver.manage().getCookieNamed("Geo").getValue());
-    }
-
+  @FindBy(css = "body > div:nth-child(5) > div > div")
+  WebElement modal;
+
+  @FindBy(css = "div._36mdTyonPU0bxbU70dDO4f")
```

Comment:
Are the classes like `_36mdTyonPU0bxbU70dDO4f` or `_3mGMvAUi8MtSBaB1BUvLtz` always the same? Yesterday there was a new release of `tracking-opt-in` and I wonder if it didn't change the class names.

Reply:
I have this same classes for about 6 days it looks like they do not change